From b592abe407a1c201c78372b1215aa0339c5bdb80 Mon Sep 17 00:00:00 2001
From: Shawn Iwinski <siwinski@redhat.com>
Date: Thu, 5 Dec 2013 15:39:53 -0500
Subject: [PATCH] Add detached HEAD detection for "* (detached from "

---
 lib/Gitter/Repository.php | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/lib/Gitter/Repository.php b/lib/Gitter/Repository.php
index 7b18d50..d2ed630 100644
--- a/lib/Gitter/Repository.php
+++ b/lib/Gitter/Repository.php
@@ -169,9 +169,10 @@ public function getBranches()
             return $branches;
         }
 
-        // Since we've stripped whitespace, the result "* (no branch)"
-        // that is displayed in detached HEAD state becomes "(nobranch)".
-        if ($branches[0] === "(nobranch)") {
+        // Since we've stripped whitespace, the result "* (detached from "
+        // and "* (no branch)" that is displayed in detached HEAD state
+        // becomes "(detachedfrom" and "(nobranch)" respectively.
+        if ((strpos($branches[0], '(detachedfrom') === 0) || ($branches[0] === '(nobranch)')) {
             $branches = array_slice($branches, 1);
         }
 
@@ -191,7 +192,7 @@ public function getCurrentBranch()
 
         foreach ($branches as $branch) {
             if ($branch[0] === '*') {
-                if ($branch === '* (no branch)') {
+                if ((strpos($branch, '* (detached from ') === 0) || ($branch === '* (no branch)')) {
                     return NULL;
                 }
 
-- 
1.8.5.1

