diff --git a/tests/Dissect/Parser/LALR1/ArithGrammar.php b/tests/Dissect/Parser/LALR1/ArithGrammar.php
index 51a75f2..5a06c5e 100644
--- a/tests/Dissect/Parser/LALR1/ArithGrammar.php
+++ b/tests/Dissect/Parser/LALR1/ArithGrammar.php
@@ -39,7 +39,7 @@ class ArithGrammar extends Grammar
             })
 
             ->is('(', 'Additive', ')')
-            ->call(function ($_, $e, $_) {
+            ->call(function ($_, $e, $_2) {
                 return $e;
             });
 
