diff --git a/src/Cilex/Provider/JmsSerializerServiceProvider.php b/src/Cilex/Provider/JmsSerializerServiceProvider.php
index fa3dd0a8..656969ad 100644
--- a/src/Cilex/Provider/JmsSerializerServiceProvider.php
+++ b/src/Cilex/Provider/JmsSerializerServiceProvider.php
@@ -28,17 +28,8 @@ class JmsSerializerServiceProvider implements ServiceProviderInterface
      */
     public function register(Application $app)
     {
-        $vendorPath = isset($app['composer.vendor_path'])
-            ? $app['composer.vendor_path']
-            : __DIR__ . '/../../../vendor';
-
-        $serializerPath = $vendorPath . '/jms/serializer/src';
-        if (!file_exists($serializerPath)) {
-            $serializerPath = __DIR__ . '/../../../../../jms/serializer/src';
-        }
-
         $app['serializer.annotations'] = array(
-            array('namespace' => 'JMS\Serializer\Annotation', 'path' => $serializerPath)
+            array('namespace' => 'JMS\Serializer\Annotation', 'path' => '__PHPDIR__')
         );
 
         $app['serializer'] = $app->share(
diff --git a/src/phpDocumentor/Bootstrap.php b/src/phpDocumentor/Bootstrap.php
index cd0920a7..6a8ffcfb 100644
--- a/src/phpDocumentor/Bootstrap.php
+++ b/src/phpDocumentor/Bootstrap.php
@@ -93,7 +93,7 @@ class Bootstrap
     public function createAutoloader($vendorDir = null)
     {
         if (! $vendorDir) {
-            $vendorDir = __DIR__ . '/../../vendor';
+            $vendorDir = __DIR__ . '/vendor';
         }
 
         $autoloader_location = $vendorDir . '/autoload.php';
